arches/app/templates/views/components/cards/default.htm

```diff
@@ -393,7 +393,7 @@ <h3 class="panel-title help-panel-title" style="">
             <i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': reportExpanded(), 'fa-angle-right': !reportExpanded()}, click: function () { reportExpanded(!reportExpanded()) }"></i>
             <!-- ko if: reportExpanded() -->
             <!-- ko foreach: { data: self.preview ? [card.newTile] : card.tiles, as: 'tile' } -->
-                <div class="rp-card-section" data-bind="css: {'provisional': tile.provisionaledits() !== null && tile.userisreviewer === false, 'fullyprovisional': tile.isfullyprovisional()}">
```

Comment:
@whatisgalen yeah, this seems a little heavy handed to me... maybe we should remove the borders & background color and just use an `<hr>` between every tile in the list... add something like this here...
```html
<!-- ko if: $index() !== 0 --><hr><!-- /ko -->
```
(this would also mean we could remove the other additions from this PR, but we might think about styling the `hr` so that it looks nice & is obvious, but not heavy-handed)

Reply:
so I adjusted the border to just be bottom; you'll see a thick left-border commented out in my css. I think it's worth uncommenting just to see what it looks like. I erred on 'less is more', but interested in other opinions. For some reason I couldn't get an <hr> to show (used your snippet as a guide), so I fell back on just using border-bottom.
@robgaston @adamlodge

Reply:
@whatisgalen i think that `childless` is not really how we want to distinguish data here... that is, we want to create some distinction between tiles of the same card. we want to be able to create this distinction regardless of whether they have child cards or not. this is why i think that checking the tile index is a more scalable solution. I would like to see the styling that you have proposed applied in a way that doesn't respect the length of `cards`, but rather the number of tiles in the report (ie `$index` in my example or similar)

Reply:
Sounds good, I'll go ahead and change so that it's looking at index